.github/workflows/build.yaml

```diff
@@ -40,12 +40,12 @@ jobs:
     name: "Generate Protobufs"
     runs-on: "ubuntu-latest"
     steps:
-      - uses: "actions/setup-go@v2"
+      - uses: "actions/checkout@v3"
+      - uses: "actions/setup-go@v3"
         with:
           go-version: "^1.16"
-      - name: "Install local Go Protobuf plugins"
-        run: "go install github.com/envoyproxy/protoc-gen-validate@v0.6.7"
-      - uses: "actions/checkout@v2"
+      - name: "Install Go Tools"
+        run: "./install-tools.sh"
```

Comment:
Can we just inline the script here? It's already so small.

Reply:
I wanted to make it easy to install for people on the CLI too